portal-web/test/functional/com/liferay/portalweb/macros/BalloonEditor.macro

```diff
@@ -0,0 +1,33 @@
+definition {
+
+	macro addTable {
+		AssertElementPresent(locator1 = "CKEditorBalloonEditorSample#INSERT_BTN");
+
+		Click(locator1 = "CKEditorBalloonEditorSample#INSERT_BTN");
+
+		Click(locator1 = "CKEditorBalloonEditorSample#INSERT_TABLE_BTN");
+
+		Click(locator1 = "CKEditorBalloonEditorSample#CHK_BTN_TABLE");
+	}
+
+	macro clearParagraphWithBackspace {
+		TripleClick(locator1 = "CKEditorBalloonEditorSample#BODY_TEXT_HYPERLINK");
```

Comment:
BODY_TEXT_HYPERLINK seems like a pretty specific locator for a generic macro to clear paragraph. Try making this more general.

Reply:
I was trying but parametrizing it was giving me an error so I left it like this. will try again

Reply:
Ok, I think it is done :D